edc-tests/edc-dataplane/cloud-transfer-tests/src/test/java/org/eclipse/tractusx/edc/dataplane/transfer/test/AzureToAzureTest.java

```diff
@@ -93,6 +97,43 @@ void setup() {
         consumerBlobHelper = new AzureBlobHelper(AZBLOB_CONSUMER_ACCOUNT_NAME, AZBLOB_CONSUMER_ACCOUNT_KEY, azuriteContainer.getHost(), azuriteContainer.getMappedPort(AZURITE_CONTAINER_PORT));
     }
 
+    @Test 
+    void transferMultipleFile_success() {
+        var sourceContainer = providerBlobHelper.createContainer(AZBLOB_PROVIDER_CONTAINER_NAME);
+        var filesNames = new ArrayDeque<String>();
+        for (var i : IntStream.rangeClosed(1, 5).toArray()) {
+            var complement = new StringBuilder().append(PREFIX_FOR_MUTIPLE_FILES).append(i).append('_');
+            filesNames.add(complement.toString() + "" + TESTFILE_NAME);
+            providerBlobHelper.uploadBlobWithPrefix(sourceContainer, filesNames.getLast(), complement.toString());
+        }
+        consumerBlobHelper.createContainer(AZBLOB_CONSUMER_CONTAINER_NAME);
+        DATAPLANE_RUNTIME.getVault().storeSecret(AZBLOB_PROVIDER_KEY_ALIAS, AZBLOB_PROVIDER_ACCOUNT_KEY);
+        DATAPLANE_RUNTIME.getVault().storeSecret(AZBLOB_CONSUMER_KEY_ALIAS, """
+                {"sas": "%s","edctype":"dataspaceconnector:azuretoken"}
+                """.formatted(consumerBlobHelper.generateAccountSas(AZBLOB_CONSUMER_CONTAINER_NAME)));
+
+        var request = createMultipleFileFlowRequest(PREFIX_FOR_MUTIPLE_FILES);
+        var url = "http://localhost:%s/control/transfer".formatted(PROVIDER_CONTROL_PORT);
+
+
+        given().when()
+                .baseUri(url)
+                .contentType(ContentType.JSON)
+                .body(request)
+                .post()
+                .then()
+                .log().ifValidationFails()
+                .log().ifError()
```

Comment:
just keep one of those log setting, the second one in this case is a subset of the first one

Reply:
done